pkg/blocks/filestep_test.go

```diff
@@ -24,16 +24,11 @@ import (
 	"testing"
 
 	"github.com/facebookincubator/ttpforge/pkg/blocks"
-	"github.com/facebookincubator/ttpforge/pkg/logging"
 	"github.com/stretchr/testify/assert"
 
 	"gopkg.in/yaml.v3"
 )
 
-func init() {
```

Comment:
toggling here doesn't work well - since all of these `init` functions are in the same package you get different behavior depending on whether test file count is odd or even - removed